guides/v2.2/cloud/docker/docker-config.md

```diff
@@ -130,20 +130,20 @@ Continue launching your Docker environment in the default _production_ mode.
 
 1. Install Magento in your Docker environment.
 
-   -  Build Magento in the Docker container:
+   Build Magento in the Docker container:
 
-      ```bash
-      docker-compose run build cloud-build
-      ```
+   ```bash
+   docker-compose run build cloud-build
+   ```
 
```

Comment:
@jeff-matthews  Question about the lint checking-- when you have code samples in nested list items, the linter fails with this error  `MD046 Code block style`.  Is there a work around for this?

Reply:
@meker12, rule `MD046` is not enabled in the production style config, so I'm curious where you saw the error. Which build are you referring to? Neither the Jenkins nor the GitHub Actions Checks builds should fail on this rule.

I think there is a workaround though. I vaguely recall encountering something similar a while back. I'll have to do some digging. In the meantime, you should be able to use the bullets without failing production builds.

Reply:
Thanks for the info -- I just got the error on my local machine, and was trying to resolve before merging to GitHub.

Reply:
@dshevtsov, let us know if you remember a workaround for this.